views/beneficial-owner-other.html

```diff
@@ -11,73 +11,8 @@
-      <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
-      <br>
-      <p class="govuk-body">You can add more later.</p>
-      <br>
-
-      <form class="form" method="post">
-
-        {% include "includes/inputs/fields/name-input.html" %}
-
-        {% include "includes/inputs/address/principal-address-input.html" %}
-
-        {% set hintText = "For example, limited company or partnership" %}
-        {% include "includes/inputs/legal-form-input.html" %}
-
-        {% set hintText = "This is the law that the beneficial owner operates under" %}
-        {% include "includes/inputs/governing-law-input.html" %}
-
-        {% set registerHintText = "Include its jurisdiction. For example, Hong Kong or State of Delaware." %}
-        {% set includeJurisdiction = false %}
-        {% set registerInCountryFormedText = "Is the other legal entity already on a public register in the country it was formed in?" %}
-        {% set registrationNumberText = "Entity's registration number" %}
-
-        {% include "includes/inputs/fields/is-on-register-in-country-formed-input.html" %}
-
-        {% set dateText = "When did it become a beneficial owner for the overseas entity?" %}
-        {% include "includes/inputs/date/start-date-input.html" %}
-
-        {% call govukFieldset({
-          legend: {
-            text: "What is its nature of control?",
-            classes: "govuk-fieldset__legend--m",
-            isPageHeading: false
-          }
-        }) %}
-
-        {% set errorMessage = errors.beneficial_owner_nature_of_control_types if errors %}
-        {% include "includes/inputs/nature-of-control-checkboxes/beneficial-owner-nature-of-control-types.html" %}
-
-        {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-        {% include "includes/inputs/nature-of-control-checkboxes/trustees-nature-of-control-types.html" %}
-
-        {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-        {% include "includes/inputs/nature-of-control-checkboxes/non-legal-firm-nature-of-control-types.html" %}
-
-        {% endcall %}
-
-        {% set sanctionsListText = "Is it on the sanctions list?" %}
-        {% set sanctionsListHintText = "This means that it is subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
-        {% include "includes/inputs/fields/is-on-sanctions-list-input.html" %}
-
-        {% include "includes/inset-text.html" %}
-
-        {% include "includes/save-and-continue-button.html" %}
-
-        {% set removePathURL = OE_CONFIGS.BENEFICIAL_OWNER_OTHER_URL + OE_CONFIGS.REMOVE + "/" + id %}
-        {% set nameToBeRemoved = name %}
-        {% include "includes/remove-button.html" %}
-      </form>
-
-    </div>
-  </div>
-
+  {% include "includes/page-templates/beneficial-owner-other.html" %}
+  {% set removePathURL = OE_CONFIGS.BENEFICIAL_OWNER_OTHER_URL + OE_CONFIGS.REMOVE + "/" + id %}
```

Comment:
removePathURL has to be placed above `{% include "includes/page-templates/beneficial-owner-other.html" %}`, and `{% set nameToBeRemoved = name %} {% include "includes/remove-button.html" %}`, can be put in the common html file, check #660